content/docs/command-reference/params/index.md

```diff
@@ -144,6 +147,48 @@ $ dvc run -n train -d logs/ -o users.csv \
           python train.py
 ```
 
+## Examples: Print all parameters
+
+Following the previous example, we can use `dvc params diff` to list all of the
+param values available in the <abbr>workspace</abbr>:
```

Comment:
This example has no changes but I moved the one about Python param files to the bottom so it looks like this is added.